apps/check_config/priv/extensions/check_config.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Simple wrapper to run check_config
+# Relx extensions cannot be binary files but sourced bash files only
+# Also append ERTS BINDIR to PATH to run the escript
+
+PATH=$BINDIR:$PATH
```

Comment:
@dincho  Does `$SCRIPT_DIR/check_config` below fork? If so, does this line export the PATH for children?

Re-testing now.

Reply:
Apparently it worked for me now in one run, so the PATH problem worked for me (macOS 10.13.2)

Reply:
It works in my tests, I didn't wanter to pollute the environment if there is no explicit need.